bedrock/settings/base.py

```diff
@@ -537,11 +549,14 @@ def language_url_map_with_fallbacks():
 }
 
 MEDIA_URL = config("MEDIA_URL", default="/user-media/")
-MEDIA_ROOT = config("MEDIA_ROOT", default=path("media"))
```

Comment:
Question: is changing this going to break something else that we already use MEDIA_URL for which isn't apparent from the codebase? Grepping around, I don't see anything that uses `MEDIA_URL` in Bedrock code, but that doesn't mean Django is/was not using it. 

(Wagtail needs it to be able to serve image-library images, and it does so in a pretty standard-Django way)

Reply:
I need to shake that out with SRE, but we could do either: isolate user-uploaded media, or go all-in. with the current config, if we use the same bucket, then the Wagtail-managed images and docs would go into `s3/gcs://bedrock-media-bucket/user-media/` anyway I think, so there's no clash with what we add in to the root path